docs/components/DocumentationHeader.tsx

```diff
@@ -283,7 +285,12 @@ function SelectTheme() {
 
   return (
     <div css={SELECT_THEME_CONTAINER}>
+      <VisuallyHidden>
+        <label htmlFor="select-theme">Select your theme</label>
+      </VisuallyHidden>
+
```

Comment:
ℹ️ This modification allows the screen reader to provide information about the select. In the actual production state of the document, when navigating using an assistive technology, we don't get that much information:

![Navigating using the keyboard until reaching the select action to switch theme is not announced by  Voice Over](https://user-images.githubusercontent.com/3874873/119312530-c574ec80-bc72-11eb-8122-6da0432da3b2.gif)

This fix allows to get the following:

![Navigating using the keyboard until reaching the select action to switch theme is now announced by  Voice Over](https://user-images.githubusercontent.com/3874873/119312606-dde50700-bc72-11eb-8397-d8f0e9a8cd79.gif)